pyls/__main__.py

```diff
@@ -4,7 +4,10 @@
 import logging.config
 import sys
 
-import ujson as json
+try:
+    import ujson as json
+except ImportError:
```

Comment:
I think we should use `Exception` here instead of `ImportError` to catch any possible problem when importing `ujson`.